tonic/models/vic/vic.py

```diff
@@ -38,8 +43,36 @@ def _get_options(self):
         self._call_vic('-o')
         return self.stdout
 
-    def run(self, global_param, logdir=None):
-        """run VIC"""
+    def run(self, global_param, logdir=None, **kwargs):
+        """
+        Run VIC with specified global parameter file.
+
+        Parameters
+        ----------
+        global_param: str
+            Either a path to a VIC global parameter file or a multiline string
+            including VIC global parameter options.
+        logdir : str, optional
+            Path to write log files to.
+        **kwargs : key=value, optional
+            Keyword arguments to pass to the VIC executable. Valid options are:
+                mpi_proc : int
+                    Specifies number of processors for MPI (must be integer).
+                    Default is 1 processor.
+                valgrind : str or bool
+                    Specifies path to valgrind executable. If bool and True,
+                    valgrind will be used without specifying the full path.
+
+        Returns
+        --------
+        returncode : int
+            Return error code from VIC.
+
+        Examples
+        --------
+        retcode = vic.run(global_param_path, logdir=".", mpi_proc=4)
+
+        """
```

Comment:
I might be wrong, but it looks like some of these changes came from a different PR but are showing up as part of yours. the `mpi_proc=4` I remember adding to `tonic` functionality to deal with MPI changes in VIC.